src/views/partials/settings.js

```diff
@@ -151,12 +151,7 @@ export const settings = data => {
           <p>${getMessage('settings-email-limit-info', { limit })}</p>
 
           ${createEmailList(emails, breachCounts)}
-          <button
-            class='settings-add-email-button primary js-settings-add-email-opener'
-            ${emails.length >= AppConstants.MAX_NUM_ADDRESSES ? 'disabled' : ''}
-          >
-            ${getMessage('settings-add-email-button')}
-          </button>
+          <a class='button primary settings-add-email-button' href='dialog/add-email'>${getMessage('settings-add-email-button')}</a>
```

Comment:
Suggestion (blocking): We need to re-add the `disabled` attribute when the user reached the email limit.

Reply:
Great catch – I totally missed that added functionality.  Fixing this was funny – `disabled` is not allowed on `<a>`, so I ended up needing to update the dialog handler to allow `<button>` as well.  A worthy addition.

Should be fixed via https://github.com/mozilla/blurts-server/pull/2830/commits/9efba5d249d99479124743a60d6ea6001fa48e17

Reply:
Thank you for doing that and also adding the “add email flow” to the settings page.